src/components/RenderRouter/ContentItem.scss

```diff
@@ -826,10 +835,10 @@ transform:rotate(attr(data-value,deg,0);
     }
 
     .oneImageA {
-        font-size: 5vw;
+        font-size: 4vw;
```

Comment:
It's probably best not to scale a vertical element (the height of the font) with a horizontal measure. But I'm not sure, CSS is my weakest area. I think the best practices are to use `%` or `em`/`rem` for relative height adjustments,

Reply:
Agreed. I'll make a separate issue since I'll have to go through a bunch of CSS to ensure consistent font size throughout the app.